src/linux/Packaging.Linux/install-from-source.sh

```diff
@@ -116,7 +116,17 @@ fi
 eval "$(sed -n 's/^ID=/distribution=/p' /etc/os-release)"
 eval "$(sed -n 's/^VERSION_ID=/version=/p' /etc/os-release | tr -d '"')"
 case "$distribution" in
-    debian | ubuntu)
+    ubuntu)
```

Comment:
Don't we only want to rely on the Jammy/22.04 feeds in Ubuntu 22.04? What happens running this new block on downlevel Ubuntu versions?

Reply:
Yes, this was a good call. I spent a fair amount of time today playing around with this (particularly trying to make it work with `dpkg --compare-versions`), and I finally settled on this approach because (1) it works and (2) it's simple. However, I do recognize it may be a little _too_ simple, so if you know of a better way feel free to share.

I also added a small second commit to fix up the comments, which were bugging me 😁.